### PR TITLE
[Docs] Correct document errors in `build_from_source.md`  master -> 1.x 

### PR DESCRIPTION
**Pull request by linjie111**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by RangeKing** [comment]
Hi @linjie111 
Thanks for your kind PR. It seems that CLA is not signed. Could you sign the CLA so that eventually we could merge this PR after review? You can check the contents and follow the instruction in the communication box shown as below
image
image

File: docs/en/01-how-to-build/build_from_source.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Download
 
 ```shell
-git clone -b master git@github.com:open-mmlab/mmdeploy.git --recursive
+git clone -b 1.x git@github.com:open-mmlab/mmdeploy.git --recursive
```
Comment (lvhan028) [inline docs/en/01-how-to-build/build_from_source.md:6]: I think `dev-1.x` is better

Comment (RangeKing) [reply docs/en/01-how-to-build/build_from_source.md:6]: @lvhan028 https://github.com/open-mmlab/mmdeploy/blame/a62ed41f4457502fa152fe20d7b2e6de19ea92ec/docs/zh_cn/01-how-to-build/build_from_source.md?plain=1#L30

Comment (lvhan028) [reply docs/en/01-how-to-build/build_from_source.md:6]: I see

**Review by lvhan028 — APPROVED** [review]
LGTM

**Pull request merged**